sound/soc/sof/amd/acp.h

```diff
@@ -0,0 +1,38 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license. When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2021 Advanced Micro Devices, Inc. All rights reserved.
+ *
+ * Author: Ajit Kumar Pandey <AjitKumar.Pandey@amd.com>
+ */
+
+#ifndef __SOF_AMD_ACP_H
+#define __SOF_AMD_ACP_H
+
+#include "../ops.h"
```

Comment:
sorry, I really don't understand why you need ops.h here. You *might* need sof-priv.h here because of the `extern const struct snd_sof_dsp_ops sof_renoir_ops;` declaration and it would help with `struct snd_sof_dev` pointers too. But I don't see a need for ops.h here.

Reply:
agree with @lyakh

Reply:
ops.h and acp.h is used all .c file so we have included ops.h in this header (acp.h). If we remove it from here we need to include ops.h in all such .c file , what would be the advantage in doing that ?

Reply:
I think such postponed changes are easy to forget and would be much easier if dealt with immediately, but well, not a show-stopper